app/modules/tmr.c

```diff
@@ -135,20 +141,32 @@ static int tmr_now(lua_State* L){
 	return 1; 
 }
 
-// Lua: tmr.register( id, interval, mode, function )
+static int tmr_get( lua_State *L, int stack, timer_t *tmr ) {
+  if (lua_isuserdata(L, stack)) {
+    *tmr = (timer_t)luaL_checkudata(L, stack, "tmr.timer");
+    if (!*tmr) return luaL_error(L, "invalid userdata");
```

Comment:
The docs mention a long jump to terminate the C function: http://pgl.yoyo.org/luai/i/lua_error. These functions simply don't return to the calling context in case of an error.